cli/src/commands/project.rs

```diff
@@ -143,42 +76,108 @@ pub async fn status(api: &PhylumApi, matches: &ArgMatches) -> StdResult<(), Phyl
 
     project.write_stdout(pretty_print);
 
-    Ok(())
+    Ok(ExitCode::Ok)
+}
+
+/// List the projects in this account.
+async fn handle_list_projects(api: &PhylumApi, matches: &ArgMatches) -> CommandResult {
+    let group = matches.get_one::<String>("group").map(|g| g.as_str());
+    let org = matches.get_one::<String>("org").map(|o| o.as_str());
```

Comment:
What happens if the user receives a "org/group" format group name from an API call or something and then passes it as a group name to a command? It seems like we're not expecting it but it will _sometimes_ work anyway because _some_ of the commands only need the "org/group" form and we never check whether "group" is actually "org/group". Probably if the group parameter contains an organization, that should be used instead of the organization from the configuration.

Reply:
> What happens if the user receives a "org/group" format group name from an API

They don't.

> It seems like we're not expecting it but it will sometimes work anyway because some of the commands only need the "org/group" form and we never check whether "group" is actually "org/group". Probably if the group parameter contains an organization, that should be used instead of the organization from the configuration.

If it sometimes works, there's not really an issue with it? I see no point in supporting a format that we explicitly do **not** want to show to the user.

Reply:
Isn't that the format we return from the API, in which case it will also be the format returned when using the JSON output?

Reply:
It is, for now. It's never going to be returned in production and a lot of this code is going to go away in the future.

Reply:
Will it really go away? We're passing this value as a parameter when filtering the project listing.

Reply:
Yes, the combined path format should go away from the API entirely. That is the plan.